<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OSGCreatePulseDialog</class>
 <widget class="QWidget" name="OSGCreatePulseDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>380</width>
    <height>150</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Create Pulse</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout" rowstretch="1,1,1,1,1" columnstretch="1,1">
     <item row="2" column="0">
      <widget class="QLabel" name="labelType">
       <property name="text">
        <string>Type:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QComboBox" name="comboBoxType">
       <item>
        <property name="text">
         <string>Acceleration</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Way</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Velocity</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="labelResolution">
       <property name="text">
        <string>Resolution:</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0" colspan="2">
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="comboBoxResolution">
       <item>
        <property name="text">
         <string>16 khz</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>8 khz</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>4 khz</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="labelDuration">
       <property name="text">
        <string>Duration:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QDoubleSpinBox" name="doubleSpinBoxDuration">
       <property name="specialValueText">
        <string/>
       </property>
       <property name="suffix">
        <string> ms</string>
       </property>
       <property name="maximum">
        <double>399.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="labelUnit">
       <property name="text">
        <string>Unit:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QComboBox" name="comboBoxUnit">
       <item>
        <property name="text">
         <string>[G]</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>[m/sÂ²]</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
